src/addons/addonapi.h

```diff
@@ -25,6 +25,7 @@ class AddonApi final : public QQmlPropertyMap {
 
   Q_INVOKABLE void connectSignal(QObject* obj, const QString& signalName,
                                  const QJSValue& callback);
+  Q_INVOKABLE void log(const QString& message);
```

Comment:
Just curious. since there aren't any callers in this PR -- is future usage expected?

Reply:
This is mostly useful for debugging purposes. I used it extensively on the previous approach, but removed all the log calls before submitting it for review. This has been a problem before, this addon scripts are hard to debug without some way to see the logs, see: https://mozilla-hub.atlassian.net/browse/VPN-4055.